version.php

```diff
@@ -26,7 +26,7 @@
 defined('MOODLE_INTERNAL') || die();
 
 $plugin->component = 'mod_scormremote';
-$plugin->release = 2024092700;
+$plugin->release = 2024092701;
 $plugin->version = 2024092700; // Keep in lockstep with version.
```

Comment:
Just need to get CI green now. Looks like it's complaining about the version not matching release / upgrade steps.

Reply:
Was rushing, should be sorted now.